pkg/plugins/resources/remote/store_test.go

```diff
@@ -425,6 +425,57 @@ var _ = Describe("RemoteStore", func() {
 		})
 	})
 
+	Describe("Delete()", func() {
+		It("should delete the resource", func() {
+			// given
+			name := "tr-1"
+			meshName := "mesh-1"
+			store := setupStore("delete.json", func(req *http.Request) {
+				Expect(req.URL.Path).To(Equal(fmt.Sprintf("/meshes/%s/trafficroutes/%s", meshName, name)))
```

Comment:
Why does the url contain `/trafficroutes/` instead of `/traffic-routes/` ? Why is the test still successful ?